lib/widgets/action_sheet.dart

```diff
@@ -402,3 +410,27 @@ class ShareButton extends MessageActionSheetMenuItemButton {
     }
   }
 }
+
+class MarkAsUnreadButton extends MessageActionSheetMenuItemButton {
+  MarkAsUnreadButton({
+    super.key,
+    required super.message,
+    required super.messageListContext,
+    required this.narrow,
+  });
+
+  final Narrow narrow;
+
+  @override IconData get icon => Icons.mark_chat_unread_outlined;
+
+  @override
+  String label(ZulipLocalizations zulipLocalizations) {
+    return zulipLocalizations.actionSheetMarkAsUnread;
+  }
+
+  @override void onPressed(BuildContext context) async {
+    if (!context.mounted) return;
```

Comment:
What's the motivation for this line? It doesn't appear in the surrounding other button implementations.